packages/p2p/src/components/buy-sell/sort-dropdown.scss

```diff
@@ -1,6 +1,6 @@
 .sort-dropdown {
     height: 4.4rem;
-    margin: 2.4rem 0;
+    margin: 2.4rem 0 !important;
```

Comment:
Maybe we can try a different approach here. Instead of using `!important` to override margin, let's use `align-self: center`.

Reply:
The filters are wrapped inside `buy-sell__header-row`, you can put `align-items: center` and remove margin for currency-dropdown and sort-dropdown

Reply:
That will also work @farhan-nurzi-deriv. Let's avoid using `!important`.